### ISSUE TITLE
phys/module_mp_p3 vn_table, vm_table, revap_table are using constant mu_r=0?

### ISSUE THREAD

**Issue by AlbaraaKhayat** [wrf-model/WRF#1738]
The LUT for rain fallspeed and ventilation parameters outer loop (mu_r_loop: do ii = 1,10) is not doing any work on the parameters all entries will be the same for all ii. Is this by intent or a bug? Also most of vn, vm and revap tables are 0.0 is this also intentional?
Thank you
(line 624 in p3_init)

**Comment by hmorrison100** [comment]
@AlbaraaKhayat @dudhia Hi, yes the outer loop over rain size distribution shape parameter mu_r is not doing anything since mu_r = 0, and this is by design. Originally this was coded for variable mu_r, but we decided to use mu_r = 0 for tuning. However, we kept the outer looping to retain flexibility to easily go back to using variable mu_r in the future. The extra cost in terms of run time/storage is practically negligible. I'm not sure what you mean by most vn, vm, and revap tables are 0.0, I confirmed all entries are nonzero. Regards, Hugh
